rustls/src/server/tls13.rs

```diff
@@ -880,23 +880,37 @@ impl ExpectFinished {
         })
     }
 
-    fn emit_stateless_ticket(
+    fn emit_ticket(
         handshake: &mut HandshakeDetails,
         sess: &mut ServerSessionImpl,
         key_schedule: &KeyScheduleTraffic,
     ) {
         let nonce = rand::random_vec(32);
         let plain = get_server_session_value(handshake, key_schedule, sess, &nonce).get_encoding();
-        let ticket = match sess.config.ticketer.encrypt(&plain) {
-            Some(t) => t,
-            None => return,
+
+        let (ticket, lifetime, stateful) = if sess.config.ticketer.enabled() {
+            let ticket = match sess.config.ticketer.encrypt(&plain) {
+                Some(t) => t,
+                None => return,
```

Comment:
We should have a comment about why it is OK to return early here without forwarding the error.

Reply:
That seems orthogonal to these changes. Can you suggest some wording?

Reply:
We can do it in another PR.